app/uk/gov/hmrc/hec/repos/HECTaxCheckStore.scala

```diff
@@ -66,11 +73,16 @@ class HECTaxCheckStoreImpl @Inject() (
 ) extends HECTaxCheckStore
     with Logging {
 
-  val key: String                   = "hec-tax-check"
-  private val ggCredIdField: String = s"data.$key.taxCheckData.applicantDetails.ggCredId"
+  val key: String                      = "hec-tax-check"
+  private val ggCredIdField: String    = s"data.$key.taxCheckData.applicantDetails.ggCredId"
+  private val isExtractedField: String = s"data.$key.isExtracted"
 
   private val hecIndexes = Seq(
-    Index(Seq(ggCredIdField -> IndexType.Ascending))
+    Index(Seq("ggCredId" -> IndexType.Ascending)),
+    Index(
+      Seq("isExtracted" -> IndexType.Ascending),
+      partialFilter = Some(BSONDocument("isExtractedField" -> false))
```

Comment:
should `isExtractedField` be here without the quotes?

Reply:
whoops it should be "isExtracted" , otherwise it was creating a weird name index with very long name to changed isExtractedField to isExtracted